features/environment.py

```diff
@@ -112,6 +112,17 @@ def before_all(context: Context) -> None:
             print(" Could not check image series. It will not be used. ")
             context.config.reuse_image = None
 
+    if userdata.get("reuse_container"):
+        series = lxc_get_series(userdata.get("reuse_container"))
+        context.reuse_container = {series: userdata.get("reuse_container")}
+        print(
+            """
+           You are providing a {series} container. Make sure you are running
+           this series tests. For instance: --tags=series.{series}""".format(
```

Comment:
Would it be better if this either straight errored if the series of the container don't match the tags or auto correct the tag and run such that what you're expecting is true and just warn the user that it adapted the behavior running?

Reply:
If the series doesn't match, it won't be used. We do not have control of the behave `--tags`. it came from CLI.For instance, if the user try to run with with `--tags=a_tag_not_in_the_feature_file` so it won't run any test (they will be skipped).